tensorly/decomposition/tests/test_CMTF_ALS

```diff
@@ -0,0 +1,162 @@
+import unittest
+import tensorly as tl
+# import tensorly.random
+import cmtf_als
+import numpy as np
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
In TensorLy, we typically use `pytest `rather than `unittest`. I tend to prefer having unitest functions rather than packing it all in a big class which tends to be less readable.